File: frontend/src/app/modules/statistics/models/mental-state.ts
export class MentalState {
  readonly created: Date;
  readonly relaxation: number;
  readonly active: boolean;

  constructor(rawMentalState: Partial<MentalState>) {
    Object.assign(this, rawMentalState);
    if (rawMentalState.created) {
      this.created = new Date(rawMentalState.created);
    }
  }
}

File: unity/Assets/Scripts/models/MentalState.cs
using System.Collections.Generic;

namespace models
{
    public class MentalState
    {
        public bool Active;
        public readonly float Relaxation;

        public MentalState(float relaxation, bool active = false)
        {
            Active = active;
            Relaxation = relaxation;
        }
    }
}